<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>portForwardForm</class>
 <widget class="QWidget" name="portForwardForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="portForward_Port">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>72</y>
     <width>139</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="portForward_Message">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>110</y>
     <width>375</width>
     <height>142</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="portForward_Check">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>275</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
